PR: https://github.com/odoo/odoo/pull/190736

From: 21e162737297bbeb586d8148295a6539f2dd6c3f
From: Nguyen Dai Duong
Date: 2024-12-17 22:16:59

Breaking data model changes scores: del:3 + add:0, change matches:
-    _inherit = 'res.users'
-    target_sales_won = fields.Integer('Won in Opportunities Target')
-    target_sales_done = fields.Integer('Activities Done Target')

Total Changes: 12

[REF] {sale_}crm: remove unused fields

[1] removes the last references to `target_sales_{won,done}`.

[2] removes the last reference to `target_sales_invoiced`.

These columns have been unused for a long time so they can be safely removed.

[1] bfda483d21a035d20711e6d5916dab9df76546a6
[2] 3e39dcba5289bcaae28788a77254d65b4eb9c56c

closes odoo/odoo#190736

Related: odoo/upgrade#6935
Signed-off-by: Thibault Delavallee (tde) <tde@openerp.com>

================================= pseudo patch: =================================

--- a/addons/crm/models/__init__.py
+++ b/addons/crm/models/__init__.py
@@ -1,7 +1,6 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from . import res_users
 from . import calendar
 from . import crm_lead
 from . import crm_lost_reason

--- a/addons/crm/models/res_users.py
+++ b/None
@@ -1,11 +0,0 @@
-# -*- coding: utf-8 -*-
-# Part of Odoo. See LICENSE file for full copyright and licensing details.
-
-from odoo import fields, models
-
-
-class ResUsers(models.Model):
-    _inherit = 'res.users'
-
-    target_sales_won = fields.Integer('Won in Opportunities Target')
-    target_sales_done = fields.Integer('Activities Done Target')
